code/Demond/Python/Lab3/Lab03.py

```diff
@@ -1,8 +1,25 @@
-credit_card_list = input()
-credit_card = [int(x) for x in credit_card_list]
-print(credit_card)
-print(credit_card[: 15 : ])
-reversed_card = credit_card.reverse()
-print(reversed_card)
-# double = [int(x * 2) for x in reversed_card]
-# print(double)
+Credit_Card_Number = input() #Lines 1 - 2 will recieve user input and insert them under a list assigned to Credit Card Variable
```

Comment:
This is a variable but the style does not conform to the PEP8 style guide. https://cheatography.com/jmds/cheat-sheets/python-pep8-style-guide/